Dockerfile

```diff
@@ -0,0 +1,17 @@
+FROM openjdk:8-jre
+
+VOLUME ["/hygieia/logs"]
+
+RUN mkdir /hygieia/config
+
+EXPOSE 8080
+
+ENV PROP_FILE /hygieia/config/application.properties
+
+WORKDIR /hygieia
+
+COPY target/*.jar /hygieia/
+COPY docker/properties-builder.sh /hygieia/
+
+CMD ./properties-builder.sh &&\
```

Comment:
please implement a way so that the spring properties can be overwritten outside the docker container.

Reply:
Added in the readme notes to use -v  option